priv/templates/phx.gen.live/form.ex

```diff
@@ -0,0 +1,98 @@
+defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>Live.Form do
+  use <%= inspect context.web_module %>, :live_view
+
+  alias <%= inspect context.module %>
+  alias <%= inspect schema.module %>
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div>
+      <.header>
+        <%%= @page_title %>
+        <:subtitle>Use this form to manage <%= schema.singular %> records in your database.</:subtitle>
+      </.header>
+
+      <.simple_form for={@form} id="<%= schema.singular %>-form" phx-change="validate" phx-submit="save">
+<%= Mix.Tasks.Phx.Gen.Html.indent_inputs(inputs, 8) %>
+        <:actions>
+          <.button phx-disable-with="Saving...">Save <%= schema.human_singular %></.button>
+        </:actions>
+      </.simple_form>
+
+      <.back navigate={return_path(@return_to.key, @<%= schema.singular %>)}>Back to <%%= @return_to.name %></.back>
+    </div>
+    """
+  end
+
+  @impl true
+  def mount(_params, _session, socket) do
+    {:ok, socket}
+  end
+
+  @impl true
+  def handle_params(params, _url, socket) do
+    socket = assign(socket, :return_to, return_to(params["return_to"]))
+    {:noreply, apply_action(socket, socket.assigns.live_action, params)}
```

Comment:
I think this can all be done on mount, since it is based on params, and we don't expect the same LiveView to migrate from new/edit. Or `live_action` only works on `handle_params`?

Reply:
Great catch! Done in e52c465